mailbox/backup/src/test/java/org/apache/james/mailbox/backup/ZipAssert.java

```diff
@@ -168,6 +168,21 @@ public ZipAssert containsOnlyEntriesMatching(EntryChecks... entryChecks) throws
         }
         return myself;
     }
+    public ZipAssert containsExactlyEntriesMatching(EntryChecks... entryChecks) throws Exception {
```

Comment:
this method can reuse some part of `containsOnlyEntriesMatching()`? can you refactor it?